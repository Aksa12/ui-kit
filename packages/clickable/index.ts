export { default as Clickable } from "./components/clickable";
