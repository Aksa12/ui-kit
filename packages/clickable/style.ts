import { css } from "emotion";

export const outline = css`
  &:focus {
    outline: 0;
  }
`;

export const pointer = css`
  cursor: pointer;
`;
