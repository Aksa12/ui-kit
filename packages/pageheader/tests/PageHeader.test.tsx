import React from "react";
import { render } from "enzyme";
import toJSON from "enzyme-to-json";

import { PageHeader } from "../";
import { Clickable } from "../../clickable";

describe("PageHeader", () => {
  it("default", () => {
    const action = () => 1;
    expect(
      toJSON(
        render(
          <PageHeader
            breadcrumbElements={[
              <span key="One">One</span>,
              <span key="Two">Two</span>
            ]}
            actions={[
              <Clickable key="Action1" action={action}>
                <div>Action 1</div>
              </Clickable>
            ]}
          />
        )
      )
    ).toMatchSnapshot();
  });
});
