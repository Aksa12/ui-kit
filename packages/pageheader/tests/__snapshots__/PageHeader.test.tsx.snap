// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader default 1`] = `
<div
  class="css-jiaax5"
>
  <div
    class="css-ewxxaj"
  >
    <span>
      One
    </span>
    <span>
      Two
    </span>
  </div>
  <ul
    class="css-10pfh7o"
  >
    <li>
      <div
        class="css-1h5x3dy"
        role="button"
        tabindex="-1"
      >
        Action 1
      </div>
    </li>
  </ul>
</div>
`;
